### DISCUSSION TITLE
Update CONTRIBUTING.md to provide more guidance

### DISCUSSION THREAD

**Discussion by hanneshapke** [tensorflow/model-card-toolkit#243] [Ideas]
List out
* How to set up a dev environment
* How to test the code
* Set up Code ownership

@codesue made the great suggestion to model the `CONTRIBUTING.md` according to [tfxa](https://github.com/tensorflow/tfx-addons/blob/main/CONTRIBUTING.md).

**Comment by codesue** [comment]
Oh, I didn't mean updating the whole contributing guide. I meant including a code style section with an overview of our conventions and instructions on how to lint code locally as part of or after https://github.com/tensorflow/model-card-toolkit/pull/242, and that we could use [tfx](https://github.com/tensorflow/tfx/blob/master/CONTRIBUTING.md#running-pylint) and [tfxa](https://github.com/tensorflow/tfx-addons/blob/main/CONTRIBUTING.md#python-coding-style) as starting points.

We can take a more incremental and iterative approach, e.g. adding information about testing when that process is set up, using a separate PR for code ownership since that would require more discussion and have blocking reviewers, etc.

That said, I really like [🤗 Transformers' CONTRIBUTING.md](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md) as a general model.

**Comment by codesue** [comment]
Completed in https://github.com/tensorflow/model-card-toolkit/pull/259
